app/views/employment/EmploymentDetailsAndBenefitsView.scala.html

```diff
@@ -61,10 +61,26 @@
                 @if(benefitsDefined){
                     @messages("common.status.updated")
                 } else {
-                    @if(isInYear) {@messages("common.status.cannotUpdate")} else { @messages("common.status.notStarted")}
+                    @if(isInYear) {@messages("common.status.cannotUpdate")}  else { @messages("common.status.notStarted")}
                 }
             </span>
         </li>
+  @if(isSingleEmployment){<li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                 @if(doExpensesExist || !isInYear) {
+                    <a class="govuk-link" id="employment-expenses_link" href="@controllers.employment.routes.CheckEmploymentExpensesController.show(taxYear)">
+                    @messages("employment.expenses.label")
+                    </a>
+                } else {@messages("employment.expenses.label")}
+              </span>
+              <span class="hmrc-status-tag">
+                 @if(doExpensesExist){
+                  @messages("common.status.updated")
+                }else{
+                if(isInYear) {@messages("common.status.cannotUpdate")} else {@messages("common.status.notStarted")}
```

Comment:
I think this if is missing an @ before it, meaning the line item is being displayed incorrectly as you can see in the image comment